### PR TITLE
fix: max-height for Collapsible component

### PR DESCRIPTION
**Pull request by gtangelo**

### CODE REVIEW

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -26,8 +26,8 @@ const App = () => {
   return (
     <ThemeProvider theme={theme === 'light' ? lightTheme : darkTheme}>
       <GlobalStyles />
-      <Suspense fallback={<PageLoading />}>
-        <ErrorBoundary>
```
Comment (Peedee2002) [inline frontend/src/App.tsx:30]: NOOOO dont tell me this was the bug

**Review by Peedee2002 — APPROVED** [review]

File: frontend/src/App.tsx (hunk 1 above)
Comment (Peedee2002) [reply frontend/src/App.tsx:30]: https://github.com/csesoc/Circles/commit/384c237ca51d3a812030a91dca4773a2ff4c1b87
😳

Comment (Peedee2002) [reply frontend/src/App.tsx:30]: lmao its official that every *single* commit to dev jared made that day introduced a bug

File: frontend/src/App.tsx [hunk 2]
```diff
@@ -39,8 +39,8 @@ const App = () => {
               <Route path="*" element={<Page404 />} />
```
Comment (Peedee2002) [inline frontend/src/App.tsx:39]: if we are looking at this, one thing that annoys me is that 404 is rendered if it loads first, which is irritating. can we improve this routing to not have that happen?

File: frontend/src/App.tsx (hunk 1 above)
Comment (gtangelo) [reply frontend/src/App.tsx:30]: lmaooo i actually dont know if this is the bug or not. but like i dont think it makes sense for the error boundary to be lazy loaded coz its suppose to catch errors. will probs have to monitor this still tbh

File: frontend/src/App.tsx (hunk 2 above)
Comment (gtangelo) [reply frontend/src/App.tsx:39]: yep will fix this eventually haha

Comment (Peedee2002) [reply frontend/src/App.tsx:39]: https://stackoverflow.com/questions/54625911/404-page-in-react-renders-in-every-page I think this might help?

**Pull request merged**